src/core/connectors.js

```diff
@@ -2299,7 +2299,7 @@ const connectors = [{
 		'*://*wbru.com/*',
 		'*://embed.radio.co/player/*',
```

Comment:
This is no longer needed.

Reply:
@hornbuckle would this work still? I'm not 100% on how `allFrames: true` works, but wouldn't that need to be added to the radio.co connector entry?

Reply:
As far as I am aware, `allFrames` injects the connector script into all iframes on the page, so it doesn't need a specific URL pattern to work.

Reply:
@hornbuckle Just testing my own dev build, if I remove the WBRU entry in connectors.js it doesn't work. If I change the Radio.co entry to add WBRU as a URL and add the allFrame line it works. 

```
	label: 'Radio.co',
	matches: [
		'*://embed.radio.co/player/*',
		'*://*wbru.com/*',
	],
	js: 'connectors/radioco.js',
	id: 'radioco',
	allFrames: true,
```

Is this the approach we should go with, stacking the radio.co URLs in that entry instead of having individual connector entries? Or is it more valid to have individual entries per website that link to radioco.js? Is there analytics that track what connectors are being used, so it might be worthwhile identifying which site is doing the scrobbling?

Reply:
Oh, so, you don't need to make any changes to the existing Radio.co entry. That should stay as-is.

All you need to do is add the new WBRU entry to connectors.js, as follows, and you're done:
```
	label: 'WBRU',
	matches: [
		'*://www.wbru.com/*',
	],
	js: 'connectors/radioco.js',
	id: 'wbru',
	allFrames: true,
```

Reply:
@hornbuckle Ok, that makes sense. I thought you were saying to get rid of it entirely, my bad.

Reply:
Ah, sorry, I see it was a bit confusing because my second comment included both lines when I only meant for it to refer to the line directly above it.